.tekton/scanner-db-slim-build.yaml

```
@@ -8,16 +8,9 @@ metadata:
     build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
     pipelinesascode.tekton.dev/max-keep-runs: "500"
-    # TODO(ROX-21073): re-enable for all PR branches
     pipelinesascode.tekton.dev/on-cel-expression: |
-      (
+      (event == "pull_request" && body.action != "ready_for_review") || (
         event == "push" && target_branch.matches("^(master|release-.*|refs/tags/.*)$")
-      ) || (
-        event == "pull_request" && (
-          target_branch.startsWith("release-") ||
-          source_branch.matches("(konflux|renovate|appstudio|rhtap)") ||
-          body.pull_request.labels.exists(l, l.name == "konflux-build")
-        )
       )
```

Formatting it like this should be more consistent:
```suggestion
      (event == "pull_request" && body.action != "ready_for_review") ||
      (event == "push" && target_branch.matches("^(master|release-.*|refs/tags/.*)$"))
```

---

Done in https://github.com/stackrox/scanner/pull/1976/commits/50d16d682a7174a3209dfdf5f7b2617354828b80, also for other pipelines